spacy_llm/backends/rest/anthropic.py

```diff
@@ -0,0 +1,121 @@
+            "claude-v1.0": Endpoints.COMPLETIONS.value,
+            "claude-instant-v1.1": Endpoints.COMPLETIONS.value,
+            "claude-instant-v1.1-100k": Endpoints.COMPLETIONS.value,
+            "claude-instant-v1.0": Endpoints.COMPLETIONS.value,
+        }
+
+    @property
+    def credentials(self) -> Dict[str, str]:
+        # Fetch and check the key
+        api_key = os.getenv("ANTHROPIC_API_KEY")
+        if api_key is None:
+            raise ValueError(
+                "Could not find the API key to access the Anthropic Claude API. Ensure you have an API key "
+                "set up via the Anthropic console (https://console.anthropic.com/), then make it available as "
+                "an environment variable 'ANTHROPIC_API_KEY."
+            )
+
+        # Ensure model is supported
+        model = self._config["model"]
+        if model not in self.supported_models:
+            raise ValueError(
+                f"The specified model '{model}' is not supported by the /v1/complete endpoint. "
+                f"Choices are: {sorted(list(self.supported_models))} ."
+                "(See the Anthropic API documentation: https://console.anthropic.com/docs/api/reference)"
+            )
+
+        # Set-up headers
+        headers = {"X-API-Key": api_key}
+        assert api_key is not None
+        return headers
+
+    def __call__(self, prompts: Iterable[str]) -> Iterable[str]:
+        headers = {
+            **self._credentials,
+            "Content-Type": "application/json",
+        }
+
+        api_responses: List[str] = []
+        prompts = list(prompts)
+        url = self._url if self._url else self.supported_models[self._config["model"]]
+
+        def _request(json_data: Dict[str, Any]) -> Dict[str, Any]:
+            r = self.retry(
+                call_method=requests.post,
+                url=url,
+                headers=headers,
+                json={**json_data, **self._config},
+                timeout=self._max_request_time,
+            )
+            try:
+                r.raise_for_status()
+            except HTTPError as ex:
+                res_content = srsly.json_loads(r.content.decode("utf-8"))
+                # Include specific error message in exception.
+                raise ValueError(
+                    f"Request to Anthropic API failed: {res_content.get('error', {})}"
+                ) from ex
+            response = r.json()
+
+            # c.f. https://console.anthropic.com/docs/api/errors
+            if "error" in response:
+                if self._strict:
+                    raise ValueError(f"API call failed: {response}.")
+                else:
+                    assert isinstance(prompts, Sized)
+                    return {"error": [srsly.json_dumps(response)] * len(prompts)}
+            return response
+
+        responses = [
+            _request({"prompt": f"{Speaker.HUMAN} {prompt}{Speaker.ASST}"})
+            for prompt in prompts
+        ]
```

Comment:
Agreed with Raphael that simulating the batching like you're doing, makes most sense. But perhaps you can leave a comment in the code here explaining that batching is not currently available for this endpoint, hence this solution.